index.html

```diff
@@ -2,7 +2,21 @@
 <html lang="en">
   <head>
     <meta charset="utf8" />
+<<<<<<< HEAD
```

Comment:
Please solve the merge conflicts

Reply:
I'm working on it at the moment) I used git rebase because I was working on wrong branch and now struggling with it